File: 1-two-sum/two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
       std::unordered_map<int ,int>Index;
       int n = nums.size();
       for(int i=0;i<n;i++)
       {
            int diff = target - nums[i];
            if(Index.find(diff)!=Index.end())
            {
                return{i,Index[diff]};
            }
            Index[nums[i]] = i;
       }
       return {};
    }
};

File: 1-two-sum/two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        l1 = []
        for i in range(len(nums)):
            for j in range(i+1,len(nums)):
                s = nums[i] + nums[j]
                if s == target:
                    l1.append(i)
                    l1.append(j)
                    return l1
                